src/controllers/controllers/rl_policy.py

```diff
@@ -4,45 +4,61 @@
 import numpy as np
 from cares_reinforcement_learning.util.helpers import denormalize
 from cares_reinforcement_learning.util.network_factory import NetworkFactory
+from cares_reinforcement_learning.encoders.types import AECompositeState
 from reinforcement_learning.parse_args import parse_args
+from reinforcement_learning.parse_args_from_file import parse_args_from_file
 
 def main():
     rclpy.init()
 
     env_config, _, network_config, rest = parse_args()
     
     # speed and turn limit
-    MAX_ACTIONS = np.asarray([1, 0.45])
-    MIN_ACTIONS = np.asarray([0, -0.45])
+    MAX_ACTIONS = np.asarray([2, 0.434])
+    MIN_ACTIONS = np.asarray([0, -0.434])
 
     controller = Controller('rl_policy_', env_config['car_name'], step_length=0.1)
     policy_id = 'rl'
 
-    OBSERVATION_SIZE=12
     ACTION_NUM=2
 
+    ##############################################################
+    ## TEMPORARILY OVERRIDING NETWORK CONFIG FOR TD3AE AND SACAE
+    ##############################################################
+    _,_,network_config = parse_args_from_file()
```

Comment:
If network_config is assigned here it shouldn't be receiving the variables from parse_args too